projects/canopy/src/lib/button/button-group/button-group.component.scss

```
@@ -0,0 +1,19 @@
+@import '../../../styles/mixins.scss';
+
+.lg-button-group {
```

```suggestion
.lg-btn-group {
```
Should we abbreviate button again? for consistency?